pontus_sensors/pontus_sensors/camera_republish.py

```diff
@@ -0,0 +1,39 @@
+import rclpy
```

Comment:
Add this to the top of the file: `#!/usr/bin/env python3`. The shebang is necessary since we are working a cpp package.